code/game/objects/structures/votingbox.dm

```diff
@@ -110,6 +113,11 @@
 		voted += voter_card
 		to_chat(user,"<span class='notice'>You cast your vote.</span>")
 
+/obj/structure/votebox/proc/valid_vote(obj/item/paper/I)
+	if(length(text) > VOTE_TEXT_LIMIT || findtext(text,"<h1>Voting Results:</h1><hr><ol>"))
```

Comment:
This should probably be length_char()

Reply:
It includes html tags so it's already imprecise, i guess i can change it.